deltachat-ffi/deltachat.h

```diff
@@ -1640,7 +1640,7 @@ int             dc_set_chat_profile_image    (dc_context_t* context, uint32_t ch
 /**
  * Set mute duration of a chat.
  *
- * This value can be checked by the ui upon receiving a new message to decide whether it should trigger an notification.
+ * The ui can then call dc_chat_is_muted() when receiving a new message to decide whether it should trigger an notification.
```

Comment:
I dont know what went wrong here. These are from #1493, which is already merged.

Reply:
Fetch and do interactive rebase onto `origin/master`, dropping the first commit manually.

Reply:
Ok.